src/Simulator/IncrementalRedstoneSimulator/DaylightSensorHandler.h

```diff
@@ -0,0 +1,64 @@
+
+#pragma once
+
+
+
+
+
+namespace DaylightSensorHandler
+{
+	inline unsigned char GetPowerLevel(const cChunk & Chunk, const Vector3i Position)
+	{
+		int TimeOfDay = Chunk.GetWorld()->GetTimeOfDay();
```

Comment:
Daylight sensors also respond to stuff like current skylight level, current weather right? I know we have a cChunk::GetTimeAlteredLight but I'm unsure if it's applicable here.